connect/mirror/src/main/java/org/apache/kafka/connect/mirror/DefaultConfigPropertyFilter.java

```diff
@@ -89,7 +83,7 @@ static class ConfigPropertyFilterConfig extends AbstractConfig {
 
         ConfigPropertyFilterConfig(Map<String, ?> props) {
             super(DEF, ConfigUtils.translateDeprecatedConfigs(props, new String[][]{
```

Comment:
we don't need to call `translateDeprecatedConfigs` now, since the deprecated configs are removed